tokio/src/signal/windows.rs

```diff
@@ -66,7 +70,7 @@ pub struct CtrlC {
 impl CtrlC {
     /// Receives the next signal notification event.
     ///
-    /// `None` is returned if no more events can be received by this stream.
+    /// `None` is returned if no more events can be received by the `RxFuture`.
```

Comment:
This should say "the `CtrlC`". The `RxFuture` type is an internal implementation detail.

Reply:
or "this listener"